lib/dal/src/workspace_snapshot/update.rs

```diff
@@ -21,6 +21,8 @@ pub enum Update {
         destination: NodeInformation,
         edge_kind: EdgeWeightKindDiscriminants,
     },
+    // This is not correctly named. We really only want to replace the single node, as we also
+    // generate Update entries to handle processing the rest of the subgraph.
     ReplaceSubgraph {
```

Comment:
I *believe* that postcard doesn't care about variant names, just order, so it might be safe to just rename this (I'd want to confirm that first, though)

Reply:
It doesn't. It only cares about index/order, but I also wasn't 100% confident that *everything* was *always* going through postcard.